.vagrant/machines/default/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:60fcec81-652e-4cc9-9836-f7e21122477f
```

Comment:
Hmm, so in .gitignore we should probably add `.vagrant`, but can you remove these files for now from your commit? Let me know if you need a hand doing that.